src/compiler/crystal/syntax/lexer.cr

```diff
@@ -1574,17 +1570,30 @@ module Crystal
                                if raw_number_string.to_i64?
```

Comment:
Above here, there's

`raise_value_doesnt_fit_in(negative ? Int64 : UInt64, start, pos_before_suffix) unless @token.value`

It probably shouldn't be `Int128` now

Reply:
Yes, I also mentioned that in my last comment.
I'll do that later.